### PR TITLE
docs/fct_student_assessment

### PR DESCRIPTION
**Pull request by sleblanc23**
Added a sample query for unpacking a particular score result from `fct_student_assessment.v_other_results`. Denver is interested in several of these score results for their reporting and I thought it was worth adding the sample here rather than in a Denver custom doc.

### CODE REVIEW

File: models/core_warehouse/fct_student_assessment.yml [hunk 1]
```diff
@@ -10,6 +10,19 @@ models:
       ##### Primary Key:
         `k_student_assessment` -- There is one record per student-assessment event
 
+      ##### Example of querying "other results":
```
Comment (rlittle08) [inline models/core_warehouse/fct_student_assessment.yml:13]: very picky but can we use the ```##### Example Use Cases:``` header instead, to be consistent with other docs pages?

**Review by rlittle08 — CHANGES_REQUESTED** [review]

File: models/core_warehouse/fct_student_assessment.yml (hunk 1 above)
Comment (sleblanc23) [reply models/core_warehouse/fct_student_assessment.yml]: fixed!

**Review by rlittle08 — APPROVED** [review]

File: models/core_warehouse/fct_student_assessment.yml [hunk 2]
```diff
@@ -10,6 +10,19 @@ models:
       ##### Primary Key:
         `k_student_assessment` -- There is one record per student-assessment event
 
+      ##### Example Use Cases:
+      Any score that are not included in the wide score results columns are listed in the `v_other_results` column. They can be included
```
Comment (jalvord1) [inline models/core_warehouse/fct_student_assessment.yml:14]: this is tiny, can we change to 'Any scores' or 'Any score that is'?

Comment (sleblanc23) [reply models/core_warehouse/fct_student_assessment.yml]: oops. fixed!

**Review by jalvord1 — APPROVED** [review]

**Review by ejoranlienea — APPROVED** [review]

**Pull request merged**